assets/prometheus-adapter/service-monitor.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
If we want to actually create this we need to add it to pkg/manifests otherwise lets just mask it out for now?

Reply:
sounds good, let's mask it out.